### ISSUE TITLE
[BE] | Datadog Dashboards | 1010CG 

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#36424]
## Description

The 10-10cg team is working on the Mulesoft integration with Carma/Salesforce.  One of the benefits of this initiative is an expected improvement in response time.

## High Level User Story/ies
The 10-10 Team needs access to reporting data to monitor the health of its application on a daily, weekly and monthly basis to ensure the applications are performant.

**Question** - should I open separate tickets for 10-10 CG vs 10-10EZ?

## Metrics to Track

By any given set of dates:
-  How many attempted submissions
-  How many went through to CARMA (is this the same as a success?)
-  How many were blocked by MPI
-  How many were blocked by errors (can we define possible error types here?)
-  Number of Secondary-caregiver-only applications
- Number of dropped attachments
-  API response times (need to define this further)
     - Show an average response time by month from the time "submit" event occurs until a 1) success or a 2) failure is returned
- Core stats: Daily Submission (Attempts | Successful | Blocked | Client Failures)
- Calculated stats: Daily Submission Failures = Attempts - Successful - Blocked

## Acceptance Criteria

- [ ] All team members can access and view dashboards on key performance indicators (as defined above)
- [x] Notification alerts are set up and sent to slack based on pre-defined thresholds
- [x] Screenshot of dashboard shared here

**Comment by SKasimow** [comment]
fyi.. @lihanli

**Comment by lihanli** [comment]
@SKasimow i think another ticket should be made for the 1010ez form, some of these metrics don't apply to the 1010 ez form

**Comment by lihanli** [comment]
@SKasimow what is a secondary caregiver only application? no primary caregiver and one or two secondary caregivers?

**Comment by lihanli** [comment]
according to the stats, there have not been any dropped attachments, so i'm not able to provide a dashboard for it

**Comment by lihanli** [comment]
![image](https://user-images.githubusercontent.com/830084/155024151-55b0192d-40b4-4c17-8d67-d21a97f0cc68.png)

dashboard link: https://app.datadoghq.com/dashboard/8it-wik-f5q/vsa-1010-team

**Comment by lihanli** [comment]
i added a notification alert that triggers when the api error rate goes above 25% https://app.datadoghq.com/monitors/64590805

**Comment by SKasimow** [comment]
@lihanli Awesome! Hopefully you can walk us thru at end of sprint planning :)
